### PR TITLE
#546 Added a way to build a specific 3rd party.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: util/envDynawo.sh [hunk 1]
```diff
@@ -671,14 +679,33 @@ config_3rd_party() {
 
 build_3rd_party() {
   config_3rd_party || error_exit "Error with 3rd parties configuration."
-  make -j $DYNAWO_NB_PROCESSORS_USED
+  if [ "$DYNAWO_CMAKE_GENERATOR" = "Unix Makefiles" ]; then
```
Comment (rosiereflo) [inline util/envDynawo.sh]: Please keep the behavior by default that builds everything

File: util/envDynawo.sh [hunk 2]
```diff
@@ -324,30 +324,38 @@ set_environment() {
...
+          export_var_env_force DYNAWO_BOOST_USE_STATIC=ON
+        fi
       fi
     fi
   fi
 
   if [ -d "$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/libarchive" ]; then
-    export_var_env DYNAWO_LIBARCHIVE_HOME=$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/libarchive
-    unset DYNAWO_LIBARCHIVE_HOME_DEFAULT
+    if [ ! -z "$(ls -A $DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/libarchive)" ]; then
```
Comment (rosiereflo) [inline util/envDynawo.sh]: Why not using the -d similarly to above?

File: util/envDynawo.sh (hunk 1 above)
Comment (gautierbureau) [reply util/envDynawo.sh]: It is, if you give nothing it builds everything

File: util/envDynawo.sh (hunk 2 above)
Comment (gautierbureau) [reply util/envDynawo.sh]: it checks if folder exists and is not empty (something has been installed)

**Pull request merged**